clients/unshaded/src/main/java/tachyon/client/block/BlockWorkerClientPool.java

```diff
@@ -45,9 +46,9 @@
    * @param workerAddress the worker address
    */
   public BlockWorkerClientPool(NetAddress workerAddress) {
-    // TODO(calvin): Get the capacity from configuration.
-    super(CAPACITY);
-    mExecutorService = Executors.newFixedThreadPool(CAPACITY, ThreadFactoryUtils.build(
+    super(ClientContext.getConf().getInt(Constants.USER_BLOCK_WORKER_CLIENT_THREADS));
```

Comment:
actually  this is probably better to be `USER_LOCAL_BLOCK_WORKER_CLIENT_THREADS` with string `tachyon.user.local.block.worker.client.threads`?